### PR TITLE
imp: re-export `Timestamp` & `Duration` from `ibc_proto::google::protobuf`

### PR DESCRIPTION
**Pull request by Farhad-Shabani**
## Description

This requirement emerged while implementing the `sov-celestia` light client, eliminating the need to import the same types from `tendermint_proto::google::protobuf`.

______

### PR author checklist:
- [X] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests.
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### CODE REVIEW

File: ibc-primitives/src/lib.rs [hunk 1]
```diff
@@ -25,8 +25,9 @@ pub use traits::*;
 mod types;
 pub use types::*;
 
-/// Re-export of common proto types from the `ibc-proto` crate.
+/// Re-exports google proto types and `Protobuf` trait from the `ibc-proto-rs`
+/// crate.
 pub mod proto {
-    pub use ibc_proto::google::protobuf::Any;
+    pub use ibc_proto::google::protobuf::*;
```
Comment (rnbguy) [inline ibc-primitives/src/lib.rs]: the scope of `ibc_proto::google::protobuf::*` is too broad. The structs other than `Any`, `Timestamp`, and `Duration` are for parsing protobuf files. Do we really need them? 

```suggestion
    pub use ibc_proto::google::protobuf::{Any, Duration, Timestamp};
```

**Review by rnbguy — CHANGES_REQUESTED** [review]
Asked a question to confirm if we need to re-export everything.

**Review by rnbguy — APPROVED** [review]
Great ! Thanks :pray:

File: ibc-primitives/src/lib.rs (hunk 1 above)
Comment (Farhad-Shabani) [reply ibc-primitives/src/lib.rs]: I was under the impression that a few types are exposed by `ibc_proto::google::protobuf` not all. Thank you for pointing that out 👌🏻

**Pull request merged**